### DISCUSSION TITLE
<% tp.file.cursor(1) %> not triggered when inserting a template into a note

### DISCUSSION THREAD

**Discussion by Oratoriooooo** [SilentVoid13/Templater#768] [Help]
# my setting
![image](https://user-images.githubusercontent.com/36411771/178442485-6ac9adda-16dd-4456-8c8c-6515a8c0ff66.png)
# what I did
### sql codeblock.md path: 002 Utility/Template
````
```sql
<% tp.file.cursor(1) %>
```
````
# what I want
I want cursor automatically jump into code block when I insert this codeblock template.
# issue
currently I have to manually trigger 
![image](https://user-images.githubusercontent.com/36411771/178443251-3810c8c2-4a65-4e28-bdbd-44f6334c48bb.png)
to get cursor jump to location I want.

**Comment by AB1908** [comment]
Hmm wonder if it has to do with being in a code block. I guess you could get around this with a `tp.system.prompt` although it's slightly different behavior. I also think cursor misbehaving is a known issue.

**Comment by welpdx** [comment]
Your template works as intended for me on Win 10.

What system are you using? 
Also, are you inserting it using `Alt + E` `Templater: Open Insert Template modal" not `Templates: Insert template` (from the built in templates plugin) right?
